src/modules/claims/claims.types.ts

```diff
@@ -54,6 +55,7 @@ export interface Claim {
   isRejected?: boolean;
   namespace: string;
   redirectUri?: string;
+  vp?: VerifiablePresentation;
```

Comment:
@Harasz why does the `Claim` have a `vp` property. That seems strange to me. I would think that it might have a `vc` maybe corresponding to the issued VC, but not a `vp`

Reply:
Right, I remember now thanks!